### ISSUE TITLE
many users are not familiar with how date formats are declared

### ISSUE THREAD

**Issue by gabrielareto** [Alliance-for-Tropical-Forest-Science/DataHarmonization#24]
It may be easier for the users to pick the format date from a dropdown menu that has examples. I mean, things like "12-31-1999" instead of "mm-dd-yyyy" etc. 

At some point we talked about offering a summary of the formats that we could pick from the database, but I do not remember now having seen it in the app.

This is a vulnerability of the app -- in most cases we should be able to guess the date format, but users may declare the wrong format and make a mess.

**Comment by gabrielareto** [comment]
Would multiple selection be useful? I would not be surprised if there are datasets with a mix of date formats.

**Comment by gabrielareto** [comment]
I agree we cannot solve everything and dates are something to be stored properly at the root.

However:
- we do not want to substitute non-NA content by NA in the "processed date" column. If there are problems we cannot solve or dates we cannot recognize, it is better to pass the problem than hiding it.
- the user now can only declare one and just one date format. But they may have two and equally-important formats (for example, for two censuses that were stacked). How difficult is to allow the user to declare two date formats, and use them both within the app?
- we return a sample of unique strings, for the user to remember. But we may want to do something more comprehensive, something like grouping the dates (character strings) by presence of different non-numeric characters (/, -, letters), then sample N unique strings from each group.
- we may want to check that the format declared by the user makes sense (returns non-NA in as.Date, or something like that), and warn the user before moving forward.

**Comment by ValentineHerr** [comment]
The app already has a warning if some dates are turned into NA. And the output table has the original date column.
